docs/troubleshooting.md

```diff
@@ -85,6 +85,45 @@ $ export PATH=$PATH:/path/to/mpi/bin
 $ pip install --no-cache-dir horovod
 ```
 
+2. Are MPI libraries added to `$LD_LIBRARY_PATH` or `ld.so.conf`?
+
+If you see the error message below, it means `mpicxx` was not able to load some of the MPI libraries. If you recently
+installed MPI, make sure that path to MPI libraries are present the `$LD_LIBRARY_PATH` environment variable, or in the
+`/etc/ld.so.conf` file.
+
+```
+mpicxx: error while loading shared libraries: libopen-pal.so.40: cannot open shared object file: No such file or directory
+error: mpicxx -show failed (see error below), is MPI in $PATH?
+Note: If your version of MPI has custom command to show compilation flags, please specify it via HOROVOD_MPICXX_SHOW environment variable.
```

Comment:
has custom command --> has a custom command
specify it via --> specify it with the